docs/reference/ml/ml-shared.asciidoc

```diff
@@ -557,6 +557,14 @@ Values must be greater than zero and less than or equal to 1.
 By default, this value is calculated during hyperparameter optimization.
 end::dfas-downsample-factor[]
 
+tag::dfas-early-stopping-enabled[]
+Advanced configuration option.
+Specifies whether the training process should finish if it is not finding any
+better perfoming models. If disabled, the training process can take significantly
+longer and result in a better performing model purely by random chance.
```

Comment:
Nit. Please take or leave this suggestion. If users just skim the text, they might misunderstand the original sentence by a higher chance. E.g.: "and result in a better performing model (...)". This is why I suggested this alternative wording.

```suggestion
longer and the chance of finding a better performing model is unremarkable.
```